source/user-manual/kibana-app/index.rst

```diff
@@ -2,26 +2,54 @@
 
 .. _kibana_app:
 
-Kibana app
-==========
+Wazuh Kibana plugin
+===================
 
 .. meta::
   :description: Find information about the Wazuh Kibana app, its different features, configuration reference and how to troubleshoot some of the most common problems.
 
-The Wazuh app for Kibana lets you visualize and analyze Wazuh alerts stored in Elasticsearch. You can obtain statistics per agent, search alerts and filter using different visualizations. It integrates with the Wazuh API to retrieve information about manager and agents configuration, logs, ruleset, groups and much more.
+The Wazuh Kibana plugin allows the user to visualize and analyze Wazuh alerts stored in Elasticsearch and provides control over the Wazuh environment conveniently. It integrates with the Wazuh API to retrieve information about the Wazuh managers and the Wazuh agents configuration, logs, ruleset, groups, and more. The collected security events can be presented as PDF reports.
 
-.. note:: Read our `Searching for alerts using the Wazuh app for Kibana <https://wazuh.com/blog/searching-for-alerts-using-the-wazuh-app-for-kibana//>`_ document for more information.
+The installation process of the Wazuh Kibana plugin is described in the Elastic Stack installation section for :ref:`RPM <install_kibana_app_rpm>` and :ref:`Debian <install_kibana_app_deb>` systems.
+It is available in Kibana under the Wazuh logo icon situated on the left side menu.
 
-To install the app, you can follow our Elastic Stack installation guides (for :ref:`RPM <install_kibana_app_rpm>` or :ref:`Debian <install_kibana_app_deb>` systems).
+.. thumbnail:: ../../images/kibana-app/kibana-plugin-main/wazuh-kibana-plugin-main.png
+  :align: center
+  :width: 100%
 
-This manual describes the configuration process to get it started and the different app features that you can use. In addition to this, you can find a troubleshooting and reference guide for quick access to some key solutions and configuration options.
+The Wazuh Kibana plugin top menu offers access to the following sections:
 
-.. topic:: Contents
+.. _kibana_app_overview:
 
-    .. toctree::
-        :maxdepth: 1
+Overview
+--------
 
-        connect-kibana-app
-        features/index
-        troubleshooting
-        reference/index
+This section provides access to the Wazuh security information management, threat detection and response, auditing and policy monitoring, and regulatory compliance. The triggered alerts are organized in dashboards containing easy to analyze charts and tables.
+
+Management
```

Comment:
![image](https://user-images.githubusercontent.com/3830293/81167935-60325880-8f96-11ea-8576-7bf9bb06e835.png)


As you can see in the image, it is not needed to have the 5 sections as titles or subtitles. You can go with a list. 

Overview: This section bla
Management: This section bla bla.

Reply:
![image](https://user-images.githubusercontent.com/3830293/81168121-b0111f80-8f96-11ea-8f98-6fd44e7e0c51.png)

I would prefer every link introduced with a sentence. Example:
- Agents: this section shows the overall status of the Wazuh agents and lists them allowing access to the detailed information regarding each Wazuh agent. Further details: link_to_agents_kibana_plugin_section.

Reply:
Additionally, I would like to have a dedicated "Overview" section, where we explain every card showed in Overview.